lib/origin_simulator.ex

```diff
@@ -47,7 +49,8 @@ defmodule OriginSimulator do
   end
 
   get "/*glob" do
-    serve_payload(conn)
+    recipe_route = Simulation.route(:simulation)
+    serve_payload?(conn, recipe_route, conn.request_path)
   end
 
   post "/*glob" do
```

Comment:
Do we also want to change the implementation for post to be similar to the behaviour in the get block above?

Reply:
Can do. Looking at the tests for `POST` (2 _cf._ many GET tests), more tests are also needed for get it working exactly the same way as `GET`. Do we load test with `POST`? 

Will need to change `serve_payload?` too, as per Elixir naming convention (not returning boolean). I didn't follow the convention strictly since the function name makes sense (`do_something?` and not `adjective?` such as `even?`) and Elixir does make exception for those `is_` type boolean functions..

Reply:
Replicated the GET single-route origin simulation implementation for POST as well.